go.mod

```diff
@@ -1,6 +1,8 @@
 module github.com/k0sproject/k0s
 
-go 1.23.0
+go 1.24.0
```

Comment:
Can we separate this into its own Pull Request, along with the linter fixes, since it's belonging to #5778?